packages/dataviews/src/types.ts

```diff
@@ -108,6 +112,11 @@ export type FieldTypeDefinition< Item > = {
 	 * Callback used to sort the field.
 	 */
 	sort: ( a: Item, b: Item, direction: SortDirection ) => number;
+	/**
+	 * Callback used to set the value of the field in the item.
+	 * Defaults to `{ ...item, [field.id]: value }`.
```

Comment:
```suggestion
```

I'm confused by this line, not sure what it means. I guess we can remove it (same in other instances of this file).

For clarity, it'd be nice to rename the `value` argument to `newValue` as well.